english/coffee/elephant-poop/coffee.md

```diff
@@ -0,0 +1,4 @@
+You're check the menu and notice the very expensive coffee. It's name "Black Ivory Coffee", you order a cup of it immediately.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.